dashboards/readme.md

```diff
@@ -0,0 +1,62 @@
+## Log and Monitor the Secondary-Analysis Service 
+
+In order to monitor the service's status and respond to the incidents more efficiently, we need to properly:
+- Log the events
+- Create log-based metrics
+- Create the health checks
+- Integrate the health and metrics into centralized dashboard, e.g. Grafana.
+
+HCA DCP provides a [centralized monitoring dashboard](https://github.com/HumanCellAtlas/dcp-monitoring) based on Grafana
+for us to integrate to. 
+
+The following instructions will walk through how to set up our service monitoring dashboards on DCP Grafana from the
+Google Cloud perspective.
+
+
+### 1. Grant permissions
+- **Give permission to the Grafana Data Source**
+
+    In order to let Grafana fetch data from the Google Cloud projects, you need to give the grafana service account
```

Comment:
Just to clarify, we do not have to create this Grafana service account right? It already exists in another dcp google project and it just has to be given the right level of permissions to whichever project you want to connect?

Reply:
Exactly! I could add your statement to the readme to make it clearer! Thanks

Reply:
Thanks!